tests/unit/models/test_base.py

```diff
@@ -51,8 +53,8 @@ def test_model_reference_slug_errors():
 
     expected = (
         "Bad testcraft.yaml content:\n"
-        "- bad value1 value (in field 'value1')\n"
-        "- bad value2 value (in field 'value2')"
+        "- value error, Bad value1 value (in field 'value1')\n"
```

Comment:
I think as a follow-up we'll want to get rid of this "value error," prefix, it doesn't really add anything to the user

Reply:
I'm mixed about that. If we start using `PydanticCustomError` it'll probably be more useful (and we can subclass that to provide general error categories), and already it's helpful in distinguishing type errors from value errors.